client/src/components/BackToTop/BackToTop.stories.tsx

```diff
@@ -1,18 +1,27 @@
+import { Story, Meta } from "@storybook/react";
 import React, { Fragment } from "react";
 
 import { BackToTop } from "./BackToTop";
 
-const BackToTopTemplate = ({ focusId }) => (
+interface TemplateProps {
+  focusId: string;
+}
+const BackToTopTemplate: Story<TemplateProps> = (args) => (
```

Comment:
Not seeing the need to go from destructuring out `({focusId})` to just using `(args)` is. It makes the function signature much less useful (well, I guess we have the types now, but still doesn't help when scanning the code).